content/blogposts/2023/announcing-sourcegraph-own.md

```diff
@@ -0,0 +1,55 @@
+---
+
+We recently wrote about our [vision for code ownership](https://about.sourcegraph.com/blog/our-vision-for-code-ownership) and why we think radically better code ownership tooling is needed. In short, code ownership in practice is more nuanced than a simplistic “one file is owned by one dev” model. Even for simple scenarios, current ownership tooling is insufficient outside of the basic PR review workflows. 
+
+We’re working on a radically better way to manage code ownership that’ll:
+- Be automatically evergreen, with a high signal-to-noise ratio.
+- Answer nuanced questions about ownership: who knows about code? Who is accountable for code?
+- Is natively integrated into key workflows and enterprise systems.
+<br/>
+
+Today, we’re very pleased to announce a new experimental feature, Sourcegraph Own. It’s our very first step towards our vision, laying the foundation for radically better ownership tooling.
+
+In this release, we’ve focused on letting you answer any question about existing code ownership data. Who owns files that relate to this code? Which files are unowned? Who should I contact to get this bad code pattern removed from the codebase? Answering those questions is now one click away.
+
+Sourcegraph Own ingests data from existing CODEOWNERS files and gives you a unified layer to view and search it. Alternatively, you can send your own custom format for code ownership data to Sourcegraph’s API. We’ve added [Teams](https://docs.sourcegraph.com/admin/teams) in Sourcegraph so that any team tree structure can also be represented.
+
+Let’s go through an example of a powerful workflow that Own unlocks. Assume the `Authenticator` symbol is a bad code pattern or is vulnerable and needs to be removed from the codebase. Sourcegraph lets you quickly search for occurrences in the codebase ([example query](https://sourcegraph.com/search?q=context:global+type:symbol+Authenticator%24&patternType=regexp&sm=0)).
+
+But who owns this code? Who should I be reaching out to? There’s no easy answer to that question even with CODEOWNERS files. You’d need to check the files one by one.
+
+This is where Sourcgregraph Own comes in! Add `select:file.owners()`, and you can get the list of all the owners in scope of this search query in one second. Even better, you can export this to a spreadsheet to, say, email all the owners or send this to a tracking system. Spoiler: we’re working on sending an email or raising a ticket to all owners in one click.
+
+<Video 
+  source={{
+    mp4: 'own-landing-page-3.mp4',
+    webm: 'own-landing-page-3.webm'
+  }}
+  loop={true}
+  title="An example of looking at ownership data in Sourcegraph"
+/>  
+
+Own also lets you search for things such as:
+- What files don’t have an owner? This search query can be refined at will to give only the most important hits, that’s the power of Sourcegraph search! (add query)
+- What are the golang files that this person owns? (add query)
+<br/>
```

Comment:
@malomarrec do you want to add examples here?
I don't know how you'd search the first one, but for the second, is this correct? https://sourcegraph.sourcegraph.com/search?q=context:global+repo:%5Egithub%5C.com/sourcegraph/sourcegraph%24++file:has.owner%28%40unknwon%29+lang:go&patternType=standard&sm=1&groupBy=repo